tests/test_dependencies.py

```
@@ -0,0 +1,41 @@
+
+
+@pytest.mark.asyncio
+async def test_get_current_user_uses_cache():
+    from app.api import dependencies
+
+    dependencies._user_cache = None
+
+    user = User(
+        id=uuid.uuid4(),
+        email="test@example.com",
+        password="hashed",
```

Ruff (0.14.13)

Possible hardcoded password assigned to argument: "password"

(S106)